README.rst

```diff
@@ -30,5 +30,38 @@ phenotrex
 
 End-to-end Microbial Phenotypic Trait Prediction.
 
-* Supported platforms: Linux, MacOS, Windows
-* Free software: GPLv3 license
+Installation
+------------
+
+.. code-block::
+
+    $ pip install phenotrex[fasta]
+
+Usage
+-----
+
+Phenotrex is a component of the `PhenDB`_ web server, which performs phenotypic trait prediction on
+user-uploaded metagenomic bins. To try out phenotrex with PhenDB's pre-trained and curated set of
+trait models, genomes may thus simply be `submitted to PhenDB`_.
+
+Basic Usage
+~~~~~~~~~~~
+To use a trained phenotrex model ``MY_TRAIT.pkl`` for prediction of a phenotypic trait with a
+given genome ``genome.fna``:
+
+.. code-block::
+
+    $ phenotrex predict --classifier MY_TRAIT.pkl genome.fna > predictions.tsv
+
+
+This yields a tabular file containing a prediction regarding the presence of the trait (YES or NO),
+as well as a confidence value the model ascribes to this prediction, ranging from 0.5 to 1.0.
+
+Advanced Usage
+~~~~~~~~~~~~~~
+For training, evaluation and introspection of phenotrex models on user data, please refer to the
```

Comment:
For users not particularly familiar to ML  concepts, we should stick to one word for introspection/inspection/interpretation/explanation. I'd vote for the latter.